sigar/src/main/java/io/crate/plugin/SigarPlugin.java

```diff
@@ -60,24 +60,19 @@ public String description() {
     public Settings additionalSettings() {
         if (!sigarService.sigarAvailable()) {
             LOGGER.warn("Sigar library is not available");
-            return Settings.EMPTY;
         }
-
-        Settings.Builder settingsBuilder = Settings.builder();
-        settingsBuilder.put(MonitorModule.NODE_INFO_EXTENDED_TYPE, NODE_INFO_EXTENDED_TYPE);
```

Comment:
don't get how this is working if this is not set and the logic at the MonitorModule is not changed. the class binding depends on that setting